app/Services/NewOrdersReportService.php

```diff
@@ -191,7 +204,25 @@ private  function getNoOfSalesMadeOnEachProduct($newOrdersForComputation, $total
                     'product_count' => $product->count ?? 0,
                     'percentage' => number_format(($product->count / $totalSales) * 100, 2) ?? 0,
                 ];
-            })->sortByDesc('product_count')->take(10)->values();
+            })->sortByDesc('product_count')->take($limit)->values();
```

Comment:
Cant, we a single function and pass a sort type as an argument? instead of duplicating logic